docker-compose.yml

```diff
@@ -31,14 +41,21 @@ services:
       - app
     environment:
       PORT: 34346
-  mongo:
-    container_name: mongo
+
+  db:
+    container_name: mysql
     restart: unless-stopped
-    image: mongo:latest
+    image: mysql:8.0
+    command: --default-authentication-plugin=mysql_native_password
+    environment:
+     MYSQL_DATABASE: tourneygen
+     MYSQL_USER: root
+     MYSQL_PASSWORD: 123
```

Comment:
Should these be `$MYSQL_USER` and `$MYSQL_PASSWORD`?

Reply:
Yes, Also pls review the other branch first